packages/now-node/src/helpers.ts

```diff
@@ -73,6 +73,18 @@ function status(res: NowResponse, statusCode: number): NowResponse {
   return res;
 }
 
+function redirect(
+  res: NowResponse,
+  url: string,
+  // justification: more explicit code for easy compression
+  // eslint-disable-next-line @typescript-eslint/no-inferrable-types
+  status: number = 307
```

Comment:
```suggestion
  status = 307
```

Reply:
I know that typescript has type inference implicitly but when declaring it explicitly I think it helps a better compression of the code. People who are new to TS don't know they have implicit inference.

Reply:
> I think it helps a better compression of the code

It doesn't, the types are erased after compilation.

> People who are new to TS don't know they have implicit inference.

I think its the opposite, JS users will know what [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters) look like.

Reply:
I just made the changes suggested by @TooTallNate